src/NexusMods.DataModel/Games/GameCapabilities/FolderMatchInstallerCapability/IModInstallDestination.cs

```diff
@@ -0,0 +1,100 @@
+using NexusMods.Paths;
+
+namespace NexusMods.DataModel.Games.GameCapabilities.FolderMatchInstallerCapability;
+
+/// <summary>
+/// Represents a target used for where user mods may be manually installed when using human assisted installers
+/// such as the Advanced Installer.
+/// </summary>
+public interface IModInstallDestination
+{
+    /// <summary>
+    /// GamePath to which the relative mod file paths should appended to.
+    /// </summary>
+    public GamePath DestinationGamePath { get; }
+
+    /// <summary>
+    /// List of known recognizable file extensions for direct children of the target <see cref="DestinationGamePath"/>.
+    /// NOTE: Only include file extensions that are only likely to appear at this level of the folder hierarchy.
+    /// </summary>
+    public IEnumerable<Extension> KnownValidFileExtensions { get; }
```

Comment:
This as well is not strictly necessary for the Manual installer, could be for some other installer, but then again we already have `InstallFolderTarget` which is more descriptive.

Reply:
Same as https://github.com/Nexus-Mods/NexusMods.App/pull/667#discussion_r1337129519